src/applications/personalization/profile/tests/fixtures/users/user.js

```diff
@@ -1,85 +1,89 @@
 import { mockContactInformation } from '~/platform/user/profile/vap-svc/util/local-vapsvc.js';
 
-export const mockUser = {
-  data: {
-    id: '',
-    type: 'users_scaffolds',
-    attributes: {
-      services: [
-        'facilities',
-        'hca',
-        'edu-benefits',
-        'form-save-in-progress',
-        'form-prefill',
-        'mhv-accounts',
-        'evss-claims',
-        'form526',
-        'user-profile',
-        'appeals-status',
-        'id-card',
-        'identity-proofed',
-        'vet360',
-        'evss_common_client',
-        'claim_increase',
-      ],
-      account: { accountUuid: 'c049d895-ecdf-40a4-ac0f-7947a06ea0c2' },
-      profile: {
-        email: 'vets.gov.user+36@gmail.com',
-        firstName: 'WESLEY',
-        middleName: 'WATSON',
-        lastName: 'FORD',
-        birthDate: '1986-05-06',
-        gender: 'M',
-        zip: '21122-6706',
-        lastSignedIn: '2020-07-21T00:04:51.589Z',
-        loa: { current: 3, highest: 3 },
-        multifactor: true,
-        verified: true,
-        signIn: { serviceName: 'idme', accountType: 'N/A', ssoe: true },
-        authnContext: 'http://idmanagement.gov/ns/assurance/loa/3',
+export const makeMockUser = () => {
```

Comment:
I was running into an issue with importing the old `mockUser` object into a Cypress test, making changes to some deeply nested values to the user model was what I needed, and then there was no good way to _reset_ the `mockUser` to its original state. Using a function that simply gives a fresh mock user means it's very simple to build a mock user and customize it without polluting other tests.

Reply:
the actual object structure is unchanged!